build_fftw3.bat

```diff
@@ -1,5 +1,7 @@
+SETLOCAL
```

Comment:
You can think of SETLOCAL/ENDLOCAL as being brackets in C/C++.

so you would have:
Prompt {
  var x;
  build_environment {
    var y;
    call build_xxx {
       var z;
    }
  }
}
It is true that it's not the same doing "build_xxx.bat" than doing "call build_xxx.bat", but i don't remember the details right now.